### PR TITLE
MAINT-53372 : External-registration - user added to the users as default group and then removed and added to the externals group during the registration phase

### PR DESCRIPTION
**Pull request by hakermi**
…
ISSUE: When bind the users a group to a given space and create an external user the binding report will contain the external user as it's added to the users group and then removed, This because of broadcasting the `creation_event` to the `NewUserEventListener` which adds the user to the default `users` group.
Fix: This fix should disable the broadcast event option when creation the external user also remove the unneeded removing from the users group, So it will not be added only to the external group

### CODE REVIEW

File: component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java [hunk 1]
```diff
@@ -197,13 +197,9 @@ public boolean execute(ControllerContext context) throws Exception {
                       user.setEmail(email);
                     }
                     try {
-                        organizationService.getUserHandler().createUser(user, true);// Broadcast user creation event
+                        organizationService.getUserHandler().createUser(user, false);
```
Comment (rdenarie) [inline component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java:200]: This action completly remove all users listeners. We should not remove

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java (hunk 1 above)
Comment (hakermi) [reply component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java:200]: yes but it's not needed for external users, and everything is working fine

Comment (hakermi) [reply component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java:200]: Ok, I have another solution for this.

Comment (rdenarie) [reply component/web/security/src/main/java/org/exoplatform/web/login/externalRegistration/ExternalRegistrationHandler.java:200]: I confirm that we need to broacast the event. When creating a user, you have 11 listeners registred :
image

chat, wiki, ES, ....
We cannot set broadcast to false

**Pull request closed without merging**